### ISSUE TITLE
[Accessibility] Focus after clicking the reactions

### ISSUE THREAD

**Issue by emrahcom** [jitsi/jitsi-meet#17367]
### What happened?

Our company had an accessibility scan done for Jitsi. This is the description of one problem in the accessibility report:

> When opening the reactions menu, the focus is not moved into the menu. Instead, the user has to tab through the rest of the conference menu to get to the reactions. When the menu is opened, the focus should be set into the menu. When it is closed, focus should be set onto the triggering element. 

### Platform

- [x] Chrome (or Chromium based)
- [ ] Firefox
- [ ] Safari
- [ ] Other desktop browser
- [ ] Android browser
- [ ] iOS browser
- [ ] Electron app
- [ ] Android mobile app
- [ ] iOS mobile app
- [ ] Custom app using a mobile SDK

### Browser / app / sdk version

Any browser

### Relevant log output

```shell

```

### Reproducibility

- [x] The problem is reproducible on meet.jit.si

### More details?

<img width="639" height="201" alt="Image" src="https://github.com/user-attachments/assets/31501e5c-ba80-441f-911a-293548631ea7" />
